src/inverseRobot.js

```diff
@@ -7,7 +7,31 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+
+  for (let i = 0; i < robotValues.length; i++) {
+    for (let j = 0; j < robotValues.length; j++) {
```

Comment:
It is a bad practice to use nesting loops. Think about making all the needed action in one loop